### DISCUSSION TITLE
Details on Lava Tube Exploration Stage

### DISCUSSION THREAD

**Discussion by gijoop** [anatolianroverchallenge/rscp#11] [Q&A] [answered]
Hey,
We have a few questions about the Lava Exploration Stage. We’ve studied the manual, however, there are some details that are unclear to us. I’ve attached a picture below to help you understand what I mean.
1. The colored card mentioned in section "Lava Tube Exploration Details" “covers the entire ground.” However, what will it look like exactly? Will it look like a line that starts and ends at the opposite sides of the tube (pic 2), or will it resemble a larger, colored area (pic 1)?
2. In the same section, the measurement that we need to calculate is described as “the expected spline path.” We think we get the overall idea, however, we would be grateful if you could provide more details. For instance, by “expected,” do you mean ideally centered (pic 3), the shortest, or perhaps something else (pic 4)?

We'd really appreciate your feedback on these questions.

![pic](https://github.com/anatolianroverchallenge/rscp/assets/58444488/7829856c-4a5c-4f02-93c3-40fb48f2e104)

**Comment by senceryazici** [comment, marked as answer]
Hi @gijoop 

Thanks for asking another good question. 

For your first question, the correct representation is demonstrated in pic 2. The width of this cardboard spans to the entire width of the lava tube. The length of the cardboard is not important, since we expect the teams to report the first occurrence of this color. Our team will ensure there is enough length of this cardboard, for your sensors to be able to accurately detect it.

To answer your second question, the correct representation is pic 3. Here we expect teams to answer "if you were to travel in this lava tube keeping your distance to the walls equal, what would be the total distance on your curved/spline trajectory?"

I hope this answers your question.
Best
